src/pytest/test.py

```diff
@@ -231,6 +235,16 @@ def testGet(env):
     env.assertOk(r.execute_command(
         'ft.create', 'idx', 'schema', 'foo', 'text', 'bar', 'text'))
 
+    env.expect('ft.get').error().contains("wrong number of arguments for 'ft.get' command")
```

Comment:
Please be less specific with error text. This will make it harder to refactor in the future.

Reply:
@mnunberg @MeirShpilraien 
It is so much easier to debug later when you can see where exactly in the code we failed.
It is great for coverage as well.
Let's discuss in the next weekly

Reply:
Lets leave it for now, we can fix it after the release ..